### PR TITLE
fixed grammatical errors in  README.md

### PR DESCRIPTION
**Pull request by jaynarayan-vaishnav**
made the necessary grammatical changes as mentioned in issue #52

## Changes proposed
-fixed grammatical errors.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by CodingSpecies — APPROVED** [review]
This seems more simple in my opinion.

File: profile/README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@
     <p> Here, right now on <b> your </b> screen, you are in the presence of the most inclusive <i> Open Source </i> community on the planet! </p>
     <p> This organisation is the home of <i> open source</i>! Make Pull Requests, open Issues, share ideas, help each other and make improvements!</p>
 </div>
-<h2>Suggested steps to get started</h2>
+<h2>Steps to get started</h2>
 <ul>
-    <li>1. Join the EddieHub GitHub Org by raising an <a href="https://github.com/EddieHubCommunity/support/issues/new?assignees=&amp;labels=invite+me+to+the+organisation&amp;template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization">issue</a> (you will be sent an invitation)</li>
+    <li>1. Join the EddieHub GitHub Org by raising an <a href="https://github.com/EddieHubCommunity/support/issues/new?assignees=&amp;labels=invite+me+to+the+organisation&amp;template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization">issue</a> (you will recieve an invitation)</li>
```
Comment (Panquesito7) [inline profile/README.md:12]:
```suggestion
    <li>1. Join the EddieHub GitHub Org by raising an <a href="https://github.com/EddieHubCommunity/support/issues/new?assignees=&amp;labels=invite+me+to+the+organisation&amp;template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization">issue</a> (you will receive an invitation)</li>
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (jaynarayan-vaishnav) [reply profile/README.md:12]: sorry for the spelling mistake will correct it now.

**Comment by jaynarayan-vaishnav** [comment]
@Panquesito7  made the suggested changes  as requested by you.

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant

**Comment by jaynarayan-vaishnav** [comment]
Can anyone merge this PR

**Comment by jaynarayan-vaishnav** [comment]
@eddiejaoude  can you merge this PR

**Comment by jaynarayan-vaishnav** [comment]
Thanks @schmelto 🙂

**Pull request merged**